test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -15109,11 +15109,21 @@ BOOST_AUTO_TEST_CASE(bitwise_shifting_constantinople_combined)
 					c := shl(0xd0, shl(0x40, a))
 				}
 			}
+			function shl_combined_overflow(uint a) public returns (uint c) {
+				assembly {
+					c := shl(0x01, shl(not(0x00), a))
```

Comment:
I don't think there is a need for these, just supply a large value in `callContractFunction` below.

Reply:
I don't understand - it only works for constants. Supply a large value where?

Reply:
I thought it has two inputs, sorry.